ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1873,6 +1875,12 @@ private void LoadConditionalFormatting(ConditionalFormatting conditionalFormatti
                         var dataBar = fr.Elements<DataBar>().First();
                         if (dataBar.ShowValue != null)
                             conditionalFormat.ShowBarOnly = !dataBar.ShowValue.Value;
+                        var id = fr.Descendants<DocumentFormat.OpenXml.Office2010.Excel.Id>().FirstOrDefault();
+                        if (id != null)
+                        {
+
+                            conditionalFormat.Name = id.Text;
+                        }
```

Comment:
What happens to `conditionalFormat.Name` when `id == null` ?

Reply:
When Id is null, not load databar style of Office2010.
Office2010 generate a ConditionalformattingRuleExtension with Id to associate a WorksheetExtension - ConditionalFormatting where are extensións rule formats.